src/components/Dropdown/Dropdown.tsx

```diff
@@ -298,7 +354,7 @@ export default class Dropdown extends AutoControlledComponent<
       <Icon
         name={`chevron ${isOpen ? 'up' : 'down'}`}
```

Comment:
I am seeing this now... The available names of the icons are defined in the theme. If the theme using the Dropdown doesn't have these icons, nothing will be displayed. I would propose using some unicode characters for this (we already have something in the accordion for this). My bad that in the submenu I was using the '>' character instead of this, but I really didn't like that on windows the right arrow looks like this: 
![image](https://user-images.githubusercontent.com/4512430/50596162-9abc4a80-0ea3-11e9-8db1-bf29163a7a5d.png)
The **SMALL TRIANGLE-*** characters should be sufficient for all combination that we have across the components (see the list here: https://www.w3schools.com/charsets/ref_utf_geometric.asp). Of course later, we will support the user to have some icon shorthand instead of those.

Reply:
Please see PR: https://github.com/stardust-ui/react/pull/673 for more information.

Reply:
used unicode chars. thanks!